storybook/stories/data/Size.ts

```diff
@@ -5,370 +5,21 @@ const sizeData = [
       {
```

Comment:
Deleted a lot of entries here to make examples easier to understand. Samples datasets should be default be as small as possible, and as complex as needed.